src/components/Button/tests.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
FYI using `react-jsx` in `tsconfig.json` allow us to not have this import in every jsx/tsx file.

Reply:
Interesting. I had no idea about it.